pkg/scyllaclient/ring_describer.go

```diff
@@ -18,6 +18,9 @@ type RingDescriber interface {
 	Reset(ctx context.Context)
 	// IsTabletKeyspace returns true if keyspace has tablet replication.
 	IsTabletKeyspace(keyspace string) bool
+	// ControlTabletLoadBalancing runs scyllaclient ControlTabletLoadBalancing
+	// only when there are any tablet tables detected.
```

Comment:
Please change the comment. To inform that is about enabling/disabling tablets load balancing.